### ISSUE TITLE
After login: not enough arguments for format string

### ISSUE THREAD

**Issue by vpistis** [GetTogetherComm/GetTogether#288]
Hi,

this is the error after I send data to the login page:

<img width="762" alt="Screenshot 2020-09-17 at 19 37 43" src="https://user-images.githubusercontent.com/1496665/93506981-53971700-f91d-11ea-93f5-9b827d12f9da.png">

There is a way to solve it?

Thank you!

**Comment by GrahamTheCoder** [comment]
Hi, I faced the same issue. It's because you're using python 3.8. The error shown is actually an error in formatting an error message. The root cause is actually an issue in defusedxml (which I could tell in debug mode locally with the full trace). I have successfully worked around this issue by updating defusedxml package from 0.5.0 to 0.6.0 in requirements.txt and then running 
```
virtualenv --python=python3 ./env
./env/bin/pip install -r requirements.txt
```

Would be good to get this PRed, but obviously further testing may be required.
